src/com/company/asset/AssetManager.java

```diff
@@ -74,29 +75,31 @@ public void returnBackToStore()
     {
             System.out.println("AssetNumber\tAssetName");
             assets.forEach((a) -> {
-                if (!a.isAssignedToEmployee()) {
+                if (a.getAssignedTo()==true) {
```

Comment:
a.getAssignedTo() itself can be a if expression, no need to check with true again.